content/post/practical_ml.md

```diff
@@ -0,0 +1,77 @@
+---
+author: Ilan Man
+title: "Practical tips for real-world Data Science"
+publishDate: "2018-05-15"
+draft: true
+tags: 
+ - datascience
+---
+
+Many Data Scientists today come from a hard science background - statistics, math, physics. Hard sciences have a bias towards empirical and objective truths: a correct answer exists and we can find it by employing the scientific method, usually manifested by a formulaic approach. While not a controversial statement in itself, many years of studying and application of such a mantra can collide with the realities of the Business world. There, it becomes increasingly difficult to perfectly apply the theory. As a result, the practitioner should understand how to adjust their model and their approach accordingly.
```

Comment:
I would recommend not capitalizing `Business` in this case and most cases throughout the post. I try to only capitalize titles e.g., `Data Scientists` and `Analyst` as well as departments e.g., `Marketing`. I'm torn on `Business Stakeholder`, but I would probably default to not capitalizing since it isn't a position, more a general description. 

I am totally not wedded to this though. So open to discussion

Reply:
Happy to abide.